apps/web/utils/outlook/trash.ts

```
@@ -0,0 +1,165 @@
+  } catch (error) {
+    // If the filter fails, try a different approach
+    logger.warn("Filter failed, trying alternative approach", {
+      threadId,
+      error,
+    });
+
+    try {
+      // Try to get messages by conversationId using a different endpoint
+      const messages = await client
+        .getClient()
+        .api(`/me/messages`)
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)